### PR TITLE
Delete references of chat.do action and jsp

### PR DESCRIPTION
**Pull request by lneves12**
## What does this PR change?

Delete references of the unused chat.do action and jsp.

I didn't delete the file ChatAction.java because looking at struts-config.xml it's still being used for the about and copyright page

## GUI diff

No difference.

- [ ] **DONE**

## Documentation
- No documentation needed: **add explanation. This can't be used if there is a GUI diff**
- [doc-susemanager](https://github.com/SUSE/doc-susemanager) PR or issue was created (GitHub automatic link expected below)

- [ ] **DONE**

## Test coverage
- No tests: **add explanation**
- Unit tests were added
- Cucumber tests were added

- [ ] **DONE**

## Links

Fixes https://github.com/SUSE/spacewalk/issues/8251

- [ ] **DONE**

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [x] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_lint_checkstyle"		 
- [ ] Re-run test "java_pgsql_tests"		 
- [ ] Re-run test "ruby_rubocop"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"		 
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"		 
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

File: java/code/webapp/WEB-INF/pages/help/chatindex.jsp [hunk 1]
```diff
@@ -1,16 +0,0 @@
...
-<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
-<%@ taglib uri="http://rhn.redhat.com/rhn" prefix="rhn" %>
-<%@ taglib uri="http://rhn.redhat.com/tags/list" prefix="rl" %>
-<%@ taglib uri="http://struts.apache.org/tags-bean" prefix="bean" %>
-
-<html>
-<head>
-</head>
-<body>
-    <script type="text/javascript" src="/javascript/highlander.js?cb=${rhn:getConfig('web.version')}"></script>
```
Comment (mcalmer) [inline java/code/webapp/WEB-INF/pages/help/chatindex.jsp]: Can we remove the javascript file as well? `./web/html/javascript/highlander.js`

File: java/code/webapp/WEB-INF/pages/help/chatindex.jsp [hunk 2]
```diff
@@ -1,16 +0,0 @@
...
-    <h1><rhn:icon type="header-help" title="help.jsp.chat" /> <bean:message key="help.jsp.chat" /></h1>
-    <p><bean:message key="help.jsp.chatinfo"/></p>
-    <p><bean:message key="help.jsp.chatlink"/></p>
```
Comment (mcalmer) [inline java/code/webapp/WEB-INF/pages/help/chatindex.jsp]: Can we remove the messages as well?

Comment (lneves12) [reply java/code/webapp/WEB-INF/pages/help/chatindex.jsp]: done

File: java/code/webapp/WEB-INF/pages/help/chatindex.jsp (hunk 1 above)
Comment (lneves12) [reply java/code/webapp/WEB-INF/pages/help/chatindex.jsp]: The js file we can't remove it. It's being used in 4 more places

Comment (mcalmer) [reply java/code/webapp/WEB-INF/pages/help/chatindex.jsp]: ahh, ok

**Pull request merged**